test/intl402/NumberFormat/constructor-max-and-min-exponent-limits.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-tointlmathematicalvalue
+description: Tests exponent limits
+features: [Intl.NumberFormat-v3]
+---*/
+
+const nf = Intl.NumberFormat();
+
+assert.sameValue(nf.format(-100e309), "-∞", "Should output -∞");
+assert.sameValue(nf.format(-100e-309), "-0", "Should output  -0");
+assert.sameValue(nf.format(100e309), "∞", "Should output  ∞");
```

Comment:
Suggestion: pending the final resolution on https://github.com/tc39/proposal-intl-numberformat-v3/pull/128, we should offer two valid outputs: the `"∞"` output as well as the long form `1,000,000,...,000` output.

Reply:
This comment is obsolete now. There is one deterministic result (technically two due to float64 rounding).